frontend/src/components/Widgets/CustomEntitiesDropdown.tsx

```diff
@@ -213,6 +211,7 @@ export default function CustomEntitiesDropdown({
     refreshStatement,
     statement,
     fieldName = "",
+    getPreLevelSelectedValues
   },
 }: any) {
```

Comment:
I don't dislike it, but according to the [issue specification](https://metacell.atlassian.net/browse/SCKAN-238) there should be no badge/counter when we are connecting to all the above.

>then the from input text should be empty (no chips and no counter).

![image](https://github.com/MetaCell/sckan-composer/assets/19196034/6f12c63e-9564-4293-a817-e52bdfcbb532)

Reply:
@ddelpiano I don't dislike the no removal of the badges because I think it makes it a bit more explicitly that the from entities are not empty. What do you think?

Reply:
@afonsobspinto imo we should remove the badges because this is also another / extra indicator that the from is empty meaning linking to all above